tests/conditional_statements_spec.rb

```diff
@@ -50,7 +77,17 @@
 end
 
 describe "Task 5:", host: :localhost do
```

Comment:
This task test fails for me @kjhenner. As far as I can tell, it's because the `puppet job` command has not been invoked and the `/etc/pasture_config.yaml` files are not created inside of each container.